.vscode/settings.json

```diff
@@ -11,5 +11,9 @@
   "files.trimTrailingWhitespace": true,
   "[markdown]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
-  }
+  },
+  "[python]": {
+    "editor.defaultFormatter": "ms-python.black-formatter"
+  },
+  "python.languageServer": "Pylance"
```

Comment:
Can we add

```json
"[django-html][html]": {
    "editor.defaultFormatter": "monosans.djlint",
    "djlint.enableLinting": true,
    "djlint.profile": "django"
},
"workbench.editorAssociations": {
    "*.db": "sqlite-viewer.option"
}
```

Reply:
For sure, added it in 2a430101f162061360ad05c00ba7c0dde251cb27